packages/modules-runtime/modules-runtime.js

```diff
@@ -13,10 +13,6 @@ if (typeof Profile === "function" &&
   };
 }
 
-// On the client, make package resolution prefer the "browser" field of
-// package.json files to the "main" field.
-options.browser = Meteor.isClient;
```

Comment:
@benjamn Yes, it shouldn't be reverted and the commit above is actually deceiving as I didn't end up publishing `modules-runtime` in the release process for `rc.0`.  But this should accurately reflect what was published.  I'll fix this.